src/main/kotlin/net/maizegenetics/phgv2/cli/ExportVcf.kt

```diff
@@ -127,8 +137,23 @@ class ExportVcf : CliktCommand(help = "Export given samples to an h.vcf file") {
             outputDir
         )
 
+        if (regionsFile.isNotBlank()) {
+            when(File(regionsFile).extension) {
+                "vcf" -> {
+                    val tmpFilename = Files.createTempFile("tempBedfile", ".bed").absolutePathString()
+                    writeBedfileFromVcf(regionsFile, tmpFilename)
+                    command.add("--regions-file")
+                    command.add(tmpFilename)
+                }
+                "bed" -> {
+                    command.add("--regions-file")
+                    command.add(regionsFile)
+                }
+                else -> throw IllegalArgumentException("Regions file $regionsFile must end in .bed or .vcf.")
```

Comment:
It would be easy to add a junit to hit this (in ExportVcfTest).  Send in any existing test file that doesn't end with .bed or .vcf and it will hit this exception.

Reply:
Added a test for regions file extension.